assets/disclosure.css

```diff
@@ -35,6 +26,19 @@
   box-shadow: var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
 }
 
+.disclosure__list {
+  position: relative;
+  overflow-y: auto;
+  font-size: 1.4rem;
+  padding-bottom: 0.5rem;
+  padding-top: 0.5rem;
```

Comment:
I wonder if we should have a bit more spacing to allow the focus state to come up fully when borders/border radius is used 🤔 
What do you think ?
![](https://screenshot.click/04-15-eaecw-hsouq.png)

Reply:
Looks like the full outline is always visible, minus the corner of course, which theoretically we could probably fixed with more advanced focus styles. Or do you just mean there should potentially be some white space between the outline and the border? I feel like the scrolling that happens on tab just results in that positioning sometimes so I'm not sure if we can control that with margin and padding. We could probably add a bit of `scroll-padding-top/bottom` to the list though maybe?

Reply:
Hmm good point, looks like adding `scroll-padding: 0.5rem 0;` seem to help with it 👍

Reply:
Donezo